pkg/chunk/grpc/grpc.proto

```diff
@@ -0,0 +1,122 @@
+syntax = "proto3";
+
+package grpc;
+
+import "google/protobuf/empty.proto";
+
+service grpc_store {
+    //index-client
+    rpc WriteIndex(WriteIndexRequest) returns (google.protobuf.Empty);
+    rpc QueryIndex(QueryIndexRequest) returns (stream QueryIndexResponse);
+    rpc DeleteIndex(DeleteIndexRequest) returns (google.protobuf.Empty);
+
+    //storage-client
+    rpc PutChunks(PutChunksRequest) returns (google.protobuf.Empty);
+    //In GetChunks rpc request send buf as nil
+    rpc GetChunks(GetChunksRequest) returns (stream GetChunksResponse);
+    rpc DeleteChunks(ChunkID) returns (google.protobuf.Empty);
+
+    //table-client
+    rpc ListTables(google.protobuf.Empty) returns (ListTablesResponse);
+    rpc CreateTable(TableDesc) returns (google.protobuf.Empty);
```

Comment:
I would define `CreateTableRequest` for extensibility, like we did for any other rpc. The `CreateTableRequest` can just contain `TableDesc` for now.

Reply:
Rather nesting the rpc messages. I think it's better to rename it as suggested.

Reply:
Yes, but `TableDesc` is also used in other places, no?

Reply:
Ahh... I didn't notice. TableDesc being used in many other places. I will nest the message as suggested.